crates/fluvio-spu/src/mirroring/remote/controller.rs

```diff
@@ -280,12 +286,26 @@ where
         Ok(())
     }
 
+    async fn update_status(&self, pair_status: MirrorPairStatus) -> Result<()> {
+        let now = SystemTime::now()
```

Comment:
this is repeating.  probably show move to common util

Reply:
Maybe as a method for `SharedMirrorStatusUpdate`?

Reply:
Added this to a new method in `StatusMirrorMessageSink`. what do you think?